VultisigApp/VultisigApp/Views/Vault/VaultDetailView.swift

```diff
@@ -180,6 +204,16 @@ struct VaultDetailView: View {
             .listRowInsets(EdgeInsets())
             .listRowSeparator(.hidden)
     }
+
+    private func showMonthlyReminderIfNeeded() {
+        guard !vault.isBackedUp else { return }
```

Comment:
regardless whether a vault has been backup before , we always need to remind user monthly